src/apyfixedarray.h

```diff
@@ -207,6 +212,14 @@ class APyFixedArray : public APyBuffer<mp_limb_t> {
     //! Cumulative multiplication over one or more axes, treating Nan as 0
     APyFixedArray nancumprod(std::optional<nb::int_> axis = std::nullopt) const;
 
+    // Return the maximum of an array or the maximum along an axis.
```

Comment:
```suggestion
    //! Return the maximum of an array or the maximum along an axis.
```
To make it a "doc-string"